src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -2014,12 +2016,31 @@ DDLGuard::DDLGuard(Map & map_, SharedMutex & db_mutex_, std::unique_lock<std::mu
     is_database_guard = elem.empty();
     if (!is_database_guard)
     {
+        static constexpr int MAX_TRY = 10;
+        static constexpr int INTERVAL_MS = 100;
```

Comment:
nit: `base/base/sleep.h` `:void sleepForMilliseconds(uint64_t milliseconds);` parameter is unsigned